anmykh/checkio/checkio_6.py

```diff
@@ -0,0 +1,17 @@
+# This function should take two positive numbers (length and width) as inputs and return the perimeter of a rectangle.
+import pytest
+
+
+def rectangle_perimeter(length: int, width: int) -> int:
+    perimeter = (length + width) * 2
+    return perimeter
```

Comment:
if you want, for such simple logic you can avoid redundant vriables but it is not required:



```
def rectangle_perimeter(length: int, width: int) -> int:
    return  (length + width) * 2
```

Reply:
also you could resolve this task using lambda, but it is more for fun:

```
rectangle_perimeter = lambda length, width: (length + width) * 2
```